docs/_articles/ja/apex/apex-get-started.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Quick Start Apex Development with Salesforce Extensions
+lang: en
+---
+  
+## Quick Start Apex Development
+Complete this Quick Start to create, test and debug a workflow that is created using an Apex class, and sets off a Apex trigger.
+
+We’ll create a custom object called `Book`, and add a custom field called `Price` to that custom object. We’ll update and test the updated Price field using Apex and learn how to work with Apex Code using the Salesforce Extensions.
```

Comment:
I'm having that feeling of deja vu - but I was thinking about this above - seems like it's You'll create this or that, not we, right?